client/src/app/core/core-services/active-meeting.service.ts

```diff
@@ -40,16 +42,43 @@ export class ActiveMeetingService {
         return this.meetingSubject.getValue();
     }
 
+    /**
+     * Indicating whether this service is initialized or not.
+     */
+    private isOnStartup = true;
+
     public constructor(
         private activeMeetingIdService: ActiveMeetingIdService,
         private repo: MeetingRepositoryService,
-        private autoupdateService: AutoupdateService
+        private autoupdateService: AutoupdateService,
+        private lifecycle: LifecycleService
     ) {
         this.activeMeetingIdService.meetingIdObservable.subscribe(id => {
             if (id !== undefined) {
                 this.setupModelSubscription(id);
             }
         });
+        this.lifecycle.openslidesBooted.subscribe(() => {
+            // Workaround to not fire if this service is initialized.
+            if (!this.isOnStartup) {
+                this.setupModelSubscription(this.meetingId);
+            } else {
+                this.isOnStartup = false;
+            }
```

Comment:
@FinnStutzenstein Please check again.
There is one remaining thing: I have to write this code to prevent, that the active-meeting's model-subscription is re-opened on startup. If that model-subscription is re-opened on startup, the autoupdate-service does not send any meeting-models.
I don't know if this is a bug (on client-side or on autoupdate-side) - what do you think? Do you know, what I mean?